apps/nextjs-website/react-components/components/Feature/Subtitle.tsx

```diff
@@ -11,24 +11,29 @@ const Subtitle = ({ item, theme }: FeatureStackItemProps) => {
   return (
     <Stack spacing={3} justifyContent='center' alignItems='center'>
       <Body textColor={textColor} body={item.subtitle} />
-      {(item.link !== undefined) && (
+      {item.link !== undefined && (
         <Stack
           spacing={1}
           justifyContent='center'
           alignItems='center'
           direction={'row'}
           color={theme === 'light' ? 'primary' : 'background.paper'}
         >
-          <Link
-            color='inherit'
-            href={item.link.href}
-            underline='none'
-            sx={{
-              fontWeight: 'bold',
-            }}
+          <Typography
+            component='span'
+            variant='body1'
+            color={theme === 'light' ? 'text.primary' : 'background.paper'}
+            fontSize={'18px'}
```

Comment:
```suggestion
```
I believe the `body1` variant already has a fontSize of 18 so no need to set it again.